components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/APIDefinitionFromSwagger20.java

```diff
@@ -281,51 +281,71 @@ public String createAPIDefinition(API api) throws APIManagementException {
-            infoObject.put("version", api.getId().getVersion());
-
-            //add info object to swaggerObject
-            swaggerObject.put("info", infoObject);
-
-            for (URITemplate uriTemplate : uriTemplates) {
+        JSONObject swaggerObject = new JSONObject();
+
+        //Create info object
+        JSONObject infoObject = new JSONObject();
+        infoObject.put("title", api.getId().getApiName());
+        infoObject.put("description", api.getDescription());
+
+        //Create contact object and map business owner info
+        JSONObject contactObject = new JSONObject();
+        contactObject.put("name", api.getBusinessOwner());
+        contactObject.put("email", api.getBusinessOwnerEmail());
+        //put contact object to info object
+        infoObject.put("contact", contactObject);
+
+        //Create licence object
+        JSONObject licenceObject = new JSONObject();
+
+        infoObject.put("licence", licenceObject);
+        infoObject.put("version", api.getId().getVersion());
+
+        //add info object to swaggerObject
+        swaggerObject.put("info", infoObject);
+
+        JSONObject pathsObject = new JSONObject();
+        JSONObject pathItemObject = null;
+        JSONObject operationObject;
+        JSONObject responseObject = new JSONObject();
+        //add default response
+        responseObject.put("200","{}");
+
+        for (URITemplate uriTemplate : uriTemplates) {
+            String pathName = uriTemplate.getUriTemplate();
+            if (pathsObject.get(pathName) == null) {
+                pathsObject.put(pathName, "{}");
+                pathItemObject = new JSONObject();
+            }
 
+            String httpVerb = uriTemplate.getHTTPVerb();
+            if (pathItemObject != null) {
+                operationObject = new JSONObject();
+                operationObject.put("x-auth-type", uriTemplate.getAuthType());
+                operationObject.put("x-throttling-tier", uriTemplate.getThrottlingTier());
+                operationObject.put("responses", responseObject);
+                pathItemObject.put(httpVerb.toLowerCase(), operationObject);
             }
+            pathsObject.put(pathName, pathItemObject);
+        }
 
-            //Create security scheme object
-            JSONObject securitySchemeObject = (JSONObject) parser.parse(securitySchemeObjectTemplate);
+        swaggerObject.put("paths", pathsObject);
+        swaggerObject.put("swagger", "2.0");
 
-        } catch (ParseException e) {
-            throw new APIManagementException("Error while generating swagger v2.0 resource for api " + api.getId().getProviderName()
-                    + "-" + api.getId().getApiName()
-                    + "-" + api.getId().getVersion(), e);
-        }
+        JSONObject securityDefinitionObject = new JSONObject();
+        JSONObject scopesObject = new JSONObject();
+        scopesObject.put("x-wso2-scopes", scopes);
```

Comment:
This has been fixed in the commit 8c284e54fda30c37591be7207fea99b252143945
Thanks for the comment